db/schema.rb

```diff
@@ -472,7 +475,6 @@
     t.string   "blog_link",              default: ""
     t.integer  "program_of_study",       default: 0
     t.text     "self_introduction",      default: ""
-    t.string   "slack_id",               default: ""
```

Comment:
Just wondering if you happened to know why this line was removed? The migrations introduced in this PR doesn't seem to touch this at all, so I'm not sure what caused this to happen.

Reply:
hmm thats true, i dont know how it got removed. I will add it back, thanks for spotting this